package.json

```diff
@@ -54,8 +54,9 @@
         "lint:ts": "eslint -f pretty 'packages/**/*.ts'",
         "lint:css": "stylelint 'packages/**/*.css'",
         "lint:versions": "node ./scripts/lint-versions.js",
-        "custom-element-json": "lerna exec --ignore @spectrum-web-components/styles -- wca analyze 'src/**/*.ts' --format json --outFile custom-elements.json",
-        "postcustom-element-json": "lerna exec --ignore @spectrum-web-components/styles -- node ../../scripts/add-custom-properties.js --src='custom-elements.json'",
+        "precustom-element-json": "rm packages/*/custom-elements.json",
+        "custom-element-json": "lerna exec --ignore @spectrum-web-components/styles -- wca analyze '*.d.ts' --format json --outFile custom-elements.json",
+        "postcustom-element-json": "lerna exec --ignore '{@spectrum-web-components/{bundle,,icons-ui,icons-workflow,iconset,shared,styles},example-project,swc-templates}' -- test -f custom-elements.json",
```

Comment:
Ensure that packages where a file is expected have a file before moving on, usually to `lerna-publish`.

Reply:
This _doesn't_ ensure content...not sure of a good way to do so right this minute. If anyone have a good idea, I'm all ears. If not, we can come back to that.

Reply:
Might help to have some sanity script that just checks for the presence of something you'd expect in each package, or maybe just presence and a minimum file size?

Reply:
This is an interesting approach. My first thought is to also to `test -f **/*.js` to ensure there is JS in the build?

What other things might be check for? Seems pretty ad hoc, has to be something better, but agree that anything we can do to ensure a quality publish would be nice. Let's follow up on options in #816